### ISSUE TITLE
JAWS 2024 Ver 2024.2405.83 [Salesforce] JAWS Not Announcing Content in Standard Input Fields and Text Areas

### ISSUE THREAD

**Issue by ykhurana-git** [FreedomScientific/standards-support#845]
### Summary
We have identified a bug where JAWS is not announcing the content in standard input fields and text areas. This issue is specific to JAWS, as NVDA and VoiceOver (VO) do not exhibit the same behavior.  

After conducting a thorough manual code inspection, we have not found any issues within our codebase that could be causing this problem. Therefore, we are wondering if this might be related to JAWS functionality or a specific configuration.  

Reproduction steps:
1. Navigate to salesforce.com
2. Click `Reports` tab in any cloud: Sales, Commerce etc.
3. Click `New Report` to start creating new report.
4. Select any `Report Type` (example: `Accounts`)
5. Click `Save and Run` or `Save`
6. A modal will appear asking for `Report Name`, `Report Unique Name` and `Report Description` (screenshot attached for reference)

### Expected result

`Report Description` is multi-line text area input field, for which it is expected behavior that Jaws does not read all content in these types of field.
But `Report Name` and `Report Unique Name` are standard input fields, and JAWS does not read the content of these standard input fields.

### Actual result

JAWS is not narrating the content of standard input fields.

#### JAWS version and build number
Jaws 2024.2405.83

 
#### Operating System and version
Windows 11

#### Browser and version:
Chrome Version 125.0.6422.142
